client/src/overview/components/checkmark.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Checkmark = () => {
+  return (<img
+    src='https://cdn.pixabay.com/photo/2016/10/10/01/49/hook-1727484_1280.png'
+    className='overview-styles-checkbox'
+    height='10'
+    width='10'></img>
+  );
+};
+
+export default Checkmark;
```

Comment:
feel like it is too much to use a component for checkmark. I am using the one from html ---- ✅, this can be just copy paste to the code. Maybe we can sort out one kind of checkmark and using it all the time.

Reply:
You are right, I am not using it anywhere.  I ended up putting it in productStyle component.  I have gone ahead and done what you suggested.  Thanks!